master/master.cfg

```diff
@@ -1449,11 +1445,13 @@ def prioritize_builders(buildmaster, builders):
         # releases. We care most about the most recently-released llvm so
         # that we have a full set of builds for releases, then llvm main
         # for bisection, then older llvm versions.
```

Comment:
The comment seems to imply llvm 13 should be higher priority than llvm 12, but the code does the opposite

Reply:
oops, fixed